src/pint/models/model_builder.py

```diff
@@ -283,8 +290,6 @@ def get_model(parfile):
 
     Parameters
     ----------
-    name : str
-        Name for the model.
     parfile : str
         The parfile name
```

Comment:
Expand doc to say it accepts a file-like object?

Reply:
Indeed,